doc/troubleshooting/README.md

```diff
@@ -95,7 +95,7 @@ These errors most commonly occur for one of following reasons. Please try out th
     >`Get-AppxPackage microsoft.desktopappinstaller`
 
 2. The App Execution Alias for the Windows Package Manager is disabled. You should enable the App Execution Alias for the Windows Package Manager. Go to `App execution aliases` option in `Apps & features Settings` to enable it.
-3. The App Installer did not automatically add the PATH environment variable. You should add the path environment variable. The value to add is "%userprofile%\AppData\Local\Microsoft\WindowsApps".
+3. The App Installer did not automatically add the PATH environment variable. You should add the path environment variable. The value to add is "%userprofile%\AppData\Local\Microsoft\WindowsApps". You can verify this by running `%LOCALAPPDATA%\Microsoft\WindowsApps\winget` from an administrator powershell. If the command runs then, but not before, then you very likely have a missing PATH environment component.
```

Comment:
> You can verify this by running `%LOCALAPPDATA%\Microsoft\WindowsApps\winget` from an administrator powershell. 

It should be **_For CMD  ,_** not powershell. 
For Powershell  `~\AppData\Local\Microsoft\WindowsApps\winget`

Reply:
My bad. I copied from the wrong window. I've given two examples. One for powershell familiar, one for command-prompt, and labelled as such.